packages/config/test/watch.integration.spec.ts

```
@@ -0,0 +1,61 @@
+import "../src/index.js";
+
+import {DITest, inject} from "@tsed/di";
+
+import {CONFIG_SOURCES} from "../src/constants/constants.js";
+import {type ConfigSource, withOptions} from "../src/index.js";
+
+class TestConfigSource implements ConfigSource {
+  options: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)